compiler.js

```diff
@@ -1,3 +1,76 @@
+let examples = new Map();
+
+examples["helloWorld"] = `// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+contract MyContract {
+    function helloWorld() public pure returns (string memory) {
+        return "Hello, World!";
+    }
+}`;
+
+examples["erc20"] = `// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+interface IERC20 {
```

Comment:
Maybe we could simplify this a bit be implementing only the bare minimum for erc20 (and not explicitly specifying the interface)?